### PR TITLE
Documentation for creating a Streamlit demo app

### PR DESCRIPTION
**Pull request by omri374**
Documentation for creating a Streamlit based demo for Presidio

### CODE REVIEW

File: docs/samples/deployments/index.md [hunk 1]
```diff
@@ -4,3 +4,4 @@
 - [Kubernetes](k8s/index.md)
 - [Spark/Azure Databricks](spark/index.md)
 - [Azure Data Factory](data-factory/index.md)
+- [Streamlit (simple demo app)](streamlit/index.md)
```
Comment (balteravishay) [inline docs/samples/deployments/index.md:7]: sure its a "deployment" and not "python" ?

File: docs/samples/deployments/streamlit/presidio_streamlit.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+@st.cache(allow_output_mutation=True)
+def anonymizer_engine():
+    return AnonymizerEngine()
+
+
+def get_supported_entities():
+    return analyzer_engine().get_supported_entities()
+
+
+def get_supported_anonymizers():
```
Comment (balteravishay) [inline docs/samples/deployments/streamlit/presidio_streamlit.py:27]: is this being used?

**Review by balteravishay — APPROVED** [review]
minor comments. great stuff!

File: docs/samples/deployments/index.md (hunk 1 above)
Comment (omri374) [reply docs/samples/deployments/index.md:7]: Not sure, it's neither here or there... WDYT?

File: docs/samples/deployments/streamlit/presidio_streamlit.py (hunk 2 above)
Comment (omri374) [reply docs/samples/deployments/streamlit/presidio_streamlit.py:27]: No, i'll remove it

File: docs/samples/deployments/index.md (hunk 1 above)
Comment (omri374) [reply docs/samples/deployments/index.md]: Moved to python

File: docs/samples/deployments/streamlit/presidio_streamlit.py (hunk 2 above)
Comment (omri374) [reply docs/samples/deployments/streamlit/presidio_streamlit.py]: Fixed

**Comment by omri374** [comment]
@balteravishay could you please take another look?

**Review by navalev — APPROVED** [review]

**Review by balteravishay — APPROVED** [review]

**Pull request merged**